buildSrc/src/main/kotlin/Config.kt

```diff
@@ -34,7 +34,7 @@ object Config {
             const val multidex = "androidx.multidex:multidex:2.0.1"
             const val recyclerView = "androidx.recyclerview:recyclerview:1.0.0"
 
-            const val design = "com.google.android.material:material:1.0.0"
+            const val design = "com.google.android.material:material:1.1.0-beta01"
```

Comment:
Version [1.1.0](https://github.com/material-components/material-components-android/releases/tag/1.1.0) is now stable :)

```suggestion
            const val design = "com.google.android.material:material:1.1.0"
```